app/api/files/route.ts

```
@@ -0,0 +1,94 @@
+// to create files in the browser's IndexedDB file system
+export async function POST(request: NextRequest) {
+  try {
+    const body = await request.json();
+    const { action, path, content, name, parentId, extension } = body;
+
+    console.log(`[FILES API] Action: ${action}`, { path, name, parentId, extension });
+
+    switch (action) {
+      case "write_file":
+        // For MCP compatibility - map path to name if needed
+        const fileName = name || extractFileNameFromPath(path);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)